demo/compose.rviz.yaml

```diff
@@ -0,0 +1,19 @@
+# Quick Start
+# $ xhost local:root
+# $ docker compose -f compose.rviz.yaml up
+
+services:
+
+  rviz:
+    image: husarion/rviz:noetic
+    restart: on-failure
+    network_mode: host
+    ipc: host
+    volumes:
+      - /tmp/.X11-unix:/tmp/.X11-unix:rw
+      - ./config/default.rviz:/root/.rviz2/default.rviz
```

Comment:
The empty default file should be replaced with config containing at least ouster pointcloud by default.

Reply:
Since we don't have access to the ouster, the follow-up task "add the default rviz config" will be created

Reply:
:+1: